app/components/organisms/Footer/FooterLogoCentre.tsx

```diff
@@ -2,63 +2,63 @@ import { Link, NavLink } from '@remix-run/react'
 import logo from '#app/assets/svg/icon-placeholder.svg'
 import SocialMediaButtons from '#app/components/molecules/SocialMediaButtons'
 import { type FooterProps } from './FooterBasic'
-
+import logoTest from '../../../../public/svg/logoTest.svg'
 const FooterLogoCentre = ({
-	companyName = 'CHANGE THIS TO YOUR COMPANY NAME!',
-	altText = 'Our company logo',
+  companyName = 'CHANGE THIS TO YOUR COMPANY NAME!',
+  altText = 'Our company logo',
 }: FooterProps) => {
-	return (
-		<footer className="bg-secondary lg:py-16 dark:bg-dark-secondary">
-			<div className="container">
-				<div className="flex flex-col items-center justify-around border-b border-muted-foreground/75 lg:flex-row lg:pb-8 dark:border-dark-muted-foreground/75">
-					<div
-						className="flex flex-col items-start gap-6 py-8
+  return (
+    <footer className="bg-secondary lg:py-16 dark:bg-dark-secondary">
+      <div className="container">
+        <div className="flex flex-col items-center justify-around border-b border-muted-foreground/75 lg:flex-row lg:pb-8 dark:border-dark-muted-foreground/75">
+          <div
+            className="flex flex-col items-start gap-6 py-8
 					font-bold text-secondary-foreground lg:flex-row dark:text-dark-secondary-foreground"
-					>
-						<div>
-							<NavLink to="#">Nav Label</NavLink>
-						</div>
-						<div>
-							<NavLink to="#">Nav Label</NavLink>
-						</div>
-						<div>
-							<NavLink to="#">Nav Label</NavLink>
-						</div>
-					</div>
+          >
+            <div>
+              <NavLink to="#">Nav Label</NavLink>
+            </div>
+            <div>
+              <NavLink to="#">Nav Label</NavLink>
+            </div>
+            <div>
+              <NavLink to="#">Nav Label</NavLink>
+            </div>
+          </div>
 
-					<Link to="/" className="flex w-20 items-center justify-center">
-						<img src={logo} alt={altText} />
-					</Link>
+          <Link to="/" className="flex w-48 items-center justify-center">
+            <img src={logoTest} alt={altText} />
```

Comment:
Nice work adding your custom logo to the footer and nav bar. See if you can add some custom links as well. The [tutorial](https://gedsargent.github.io/code-monster/guides/epic-news/ui/adding-links/) on Code Monster will help with this.